analytics/src/analytics/integrations/etldb/main.py

```diff
@@ -190,3 +195,26 @@ def sync_quads(db: EtlDb, dataset: EtlDataset) -> dict:
                 f"QUAD '{ghid}' title = '{quad_df['quad_name']}', row_id = {result[ghid]}",
             )
     return result
+
+
+def get_sql_file_paths() -> dict[int, str]:
+    """Get all sql files needed for database initialization."""
+    result = {}
+
+    # define the path to the sql files
+    sql_file_directory = f"{Path(__file__).resolve().parent}/migrations/versions"
+
+    # get list of sorted filenames
+    filename_list = sorted(os.listdir(sql_file_directory))
+
+    # compile result
+    pattern = re.compile(r"^\d\d\d\d_.+\.sql$")
+    for filename in filename_list:
+        # validate filename format
+        if pattern.match(filename):
+            version = int(filename[:4])
+            result[version] = f"{sql_file_directory}/{filename}"
```

Comment:
Regexes can trip some folks up (I'm pretty mediocre with them), what about something a bit more direct like:

```py
if filename.endswith(".sql"):
     version = int(filename.split("_")[0])
     ...
```

Reply:
That would fail if the filename does not include and underscore. The regex enforces the file naming convention.